packages/x-date-pickers/src/TimeField/TimeField.types.ts

```diff
@@ -48,25 +48,17 @@ export type TimeFieldProps<TEnableAccessibleFieldDOMStructure extends boolean =
        * The props used for each component slot.
        * @default {}
        */
-      slotProps?: TimeFieldSlotProps<TEnableAccessibleFieldDOMStructure>;
+      slotProps?: TimeFieldSlotProps;
     };
 
-export type TimeFieldOwnerState<TEnableAccessibleFieldDOMStructure extends boolean> =
-  TimeFieldProps<TEnableAccessibleFieldDOMStructure>;
-
 export interface TimeFieldSlots extends UseClearableFieldSlots {
   /**
    * Form control with an input to render the value.
-   * @default TextField from '@mui/material' or PickersTextField if `enableAccessibleFieldDOMStructure` is `true`.
+   * @default <PickersTextField />, or <TextField /> from '@mui/material' if `enableAccessibleFieldDOMStructure` is `false`.
```

Comment:
Not related to this PR but I noticed this inconsistent JSDoc while doing the changes